mne/viz/utils.py

```diff
@@ -2419,3 +2419,40 @@ def _handle_precompute(precompute):
                       extra='when precompute=None is used')
         precompute = dict(true=True, false=False, auto='auto')[precompute]
     return precompute
+
+
+def set_3d_axes_equal(ax):
```

Comment:
I think I would make this private :thinking: ... because I would assume we remove this "helper" once MNE requires matplotlib > 3.3